src/LabLog.Domain/Entities/Room.cs

```diff
@@ -23,6 +23,22 @@ public static Room Create(Action<ILabEvent> eventHandler)
 
         public List<Computer> Computers { get; } = new List<Computer>();
         public Guid Id { get; set; }
+        private String _name;
+        public String Name {
+            get { return _name; }
+            set
+            {
+                if (_eventHandler == null)
+                {
+                    return;
+                }
+                _name = value;
+                var @event = new LabEvent<RoomNameChangedEvent>(Id,
+                    new RoomNameChangedEvent(Name)

```

Comment:
this should really be _name rather than Name. I'm not sure you want to reference a Getter from inside a Setter